gradle-metric-schema/src/main/java/com/palantir/metric/schema/gradle/MetricSchemaPlugin.java

```diff
@@ -39,6 +39,9 @@
     public static final String GENERATE_METRICS_MARKDOWN = "generateMetricsMarkdown";
     public static final String CREATE_METRICS_MANIFEST = "createMetricsManifest";
 
+    private static final String SAFE_LOGGING_VERSION = "1.12.2";
+    private static final String TRITIUM_VERSION = "0.16.2";
```

Comment:
these end up being the minimum compatible versions, correct? can we bump tritium to `0.16.3` to pick up the error-prone annotations downgrade? the error-prone annotations 2.3.4 is causing some downstream issues I'd prefer to head off.

```suggestion
    private static final String TRITIUM_VERSION = "0.16.3";
```

Reply:
Good call, updated.